<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calcular EI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>589</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione o arquivo de temperaturas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>27</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDecimal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>,</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Separador</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSep">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Espaço</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>,</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnCalcular">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular EI</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtualizar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Opções para salvar</string>
    </property>
    <widget class="QCheckBox" name="checkBoxSVG">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar SVG</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPDF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar PDF</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCSV">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar CSV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHTML">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar HTML</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Método de cálculo</string>
    </property>
    <widget class="QRadioButton" name="radioButtonASTM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ASTM</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonEnergia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Energia Interna</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxPlotar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Plotar</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>btnAtualizar</tabstop>
  <tabstop>comboBoxSep</tabstop>
  <tabstop>comboBoxDecimal</tabstop>
  <tabstop>checkBoxPlotar</tabstop>
  <tabstop>listFiles</tabstop>
  <tabstop>radioButtonASTM</tabstop>
  <tabstop>radioButtonEnergia</tabstop>
  <tabstop>checkBoxHTML</tabstop>
  <tabstop>checkBoxCSV</tabstop>
  <tabstop>checkBoxPDF</tabstop>
  <tabstop>checkBoxSVG</tabstop>
  <tabstop>btnCalcular</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
